releases/index.html

```diff
@@ -530,6 +542,12 @@ <h3>Notebook Tool Manager 0.1.5 <em style="font-size: 0.8em; color: gray;">(Octo
                 <h2 id="wysiwyg">Rich Text Editor</h2>
                 <p>Release updates for the <a href="https://github.com/genepattern/jupyter-wysiwyg" target="_blank">Rich Text Editor</a>. Install using <em>pip install jupyter-wysiwyg</em> or <em>conda install -c genepattern jupyter-wysiwyg</em>.</p>
 
+                <h3>Rich Text Editor 19.10 <em style="font-size: 0.8em; color: gray;">(October 28, 2019)</em></h3>
+                <ul>
+                    <li>Bug fix for double-clicking erroneously displaying markdown text.</li>
+                    <li>Bug fix for markdwn cells erroneously being changed to code</li>
```

Comment:
typo - markdwn should be markdown - also need a period at the end to match the others (or take away the period in the previous line)